docs/et-api.apib

```diff
@@ -9,91 +9,171 @@ Group of all ET1-related resources.
+                    },
+                    "uuid": "4611f316-f67a-4ccc-9335-df21408e96f6"
+                },
+                {
+                    "command": "BuildRespondent",
+                    "data": {
+                        "name":"dodgy_co",
+                        "contact":"John Smith",
+                        "address_attributes": {
+                            "building":"The Shard",
+                            "street":"Downing Street",
+                            "locality":"Westminster",
+                            "county":"",
+                            "post_code":"WC1A 1AA"
+                        },
+                        "dx_number":"724060 Derby 21",
+                        "address_telephone_number":"02081234567",
+                        "alt_phone_number":"07123456789",
+                        "contact_preference":"email",
+                        "email_address":"email@domain.com",
+                        "fax_number":"02071234567",
+                        "organisation_employ_gb":100,
+                        "organisation_more_than_one_site":true,
+                        "employment_at_site_number":100
+                    },
+                    "uuid":"d80d85ee-9d5c-4c9c-bd86-aa3da851c622"
+                },
+                {
+                    "command":"BuildRepresentative",
+                    "data": {
+                        "name":"Jane Doe",
+                        "organisation_name":"Representative Org",
+                        "address_attributes": {
+                            "building":"Rep Building",
+                            "street":"Rep Street",
+                            "locality":"Rep Town",
+                            "county":"Rep County",
+                            "post_code":"WC2 2BB"
+                        },
+                        "address_telephone_number":"02077654321",
+                        "mobile_number":"07987654321",
+                        "representative_type":"Private Individual",
+                        "dx_number":"724060 Derby 21",
+                        "reference":"Rep Ref",
+                        "contact_preference":"Fax",
+                        "email_address":"rep@domain.co.uk",
+                        "fax_number":"02073456789",
+                        "disability":true,
+                        "disability_information":"Lorem ipsum disability"
+                    },
+                    "uuid":"cdb093bf-82ee-4025-88a8-d00a6717a6aa"
+                }
+            ]
         }
 
 + Response 201 (application/json)
 
         {
-          "data": {
-            "reference": "992000000100",
-            "submitted_at": "2018-01-01 12:00:00 GMT+00",
-            "pdf": "s3/link/to/form/pdf"
-          }
+            "data": {
+                "reference": "992000000100",
+                "submitted_at": "2018-01-01",
+                "pdf": "s3/link/to/form.pdf",
```

Comment:
Can you change this to pdf_url - as we are not returning the pdf itself but a url to it ?

Reply:
Updated in b07843e